website/src/components/author/index.js

```diff
@@ -28,49 +29,63 @@ function Author(props) {
     <BlogLayout title={name}>
       <Head>
         <meta property="og:type" content="profile" />
-        <meta property="og:title" content={`${name} - ${blogData && blogData.blogTitle && blogData.blogTitle}`} />
-        {description &&
+        <meta
+          property="og:title"
+          content={`${name} - ${
+            blogData && blogData.blogTitle && blogData.blogTitle
+          }`}
+        />
+        {description && (
           <meta property="og:description" content={`${description}`} />
-        }
+        )}
       </Head>
 
-      <main
-        itemScope
-        itemType="http://schema.org/Person">
+      <main itemScope itemType="http://schema.org/Person">
         <section className="author-header row align-items-center">
           <div className="author-header-left">
-            <img src={imageCacheWrapper(image_url)} alt={name} itemProp="image" />
+            <img
+              src={imageCacheWrapper(image_url)}
+              alt={name}
+              itemProp="image"
+            />
           </div>
           <div className="author-header-right">
             <h1 itemProp="name">{name}</h1>
             <h4 className="author-title" itemProp="jobTitle">
-              {job_title && job_title} {organization && `@ ${organization}`} 
+              {job_title && job_title} {organization && `@ ${organization}`}
               <div className="author-links">
-              {links && links.length > 0 && (
-                <>
-                <span>|</span>
-                {links.map((link, i) => (
-                  <a 
-                    href={link.url} 
-                    title={`${name} - Social`} 
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    key={i}
-                  >
-                    <i className={`fab ${link.icon}`}></i>
-                  </a>
-                ))}
-                </>
-              )
-              }
-            </div>
+                {links && links.length > 0 && (
+                  <>
+                    <span>|</span>
+                    {links.map((link, i) => (
+                      <a
+                        href={link.url}
+                        title={`${name} - Social`}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        key={i}
+                      >
+                        {/* <!--! Font Awesome Pro 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --> */}
+                        {link?.icon ? (
+                          <div className="social-icon">
+                            {getSvgIcon(link?.icon)}
+                          </div>
```

Comment:
Now uses svgs for author social links: https://docs-getdbt-com-git-remove-fontawesome-dbt-labs.vercel.app/author/emily_riederer